### PR TITLE
[clang] Fix OMPT ident flag in combined distribute parallel for pragma

### PR DESCRIPTION
**Pull request by mikaoP**

### CODE REVIEW

**Comment by mikaoP** [comment]
@alexey-bataev is it okay to integrate this?

File: clang/lib/CodeGen/CodeGenFunction.h [hunk 1]
```diff
@@ -3812,6 +3812,8 @@ class CodeGenFunction : public CodeGenTypeCache {
     Expr *NextLB = nullptr;
     /// Update of UB after a whole chunk has been executed
     Expr *NextUB = nullptr;
+    /// Distinguish between the for distribute and sections
+    OpenMPDirectiveKind DKind = llvm::omp::OMPD_unknown;
```
[the remark this answers is not in the document]
Comment (mikaoP) [reply clang/lib/CodeGen/CodeGenFunction.h:3816]: https://github.com/llvm/llvm-project/pull/80987/files#diff-85a81ef894934d95e1d7917fd1d63a60f9ba05b978f5f29815596636dc134be5R3084 here for example

**Review by alexey-bataev — APPROVED** [review]
LG

**Comment by mikaoP** [comment]
Tests passing now

**Comment by mikaoP** [comment]
@alexey-bataev Is it good now?

**Comment by alexey-bataev** [comment]
Yes, as it was before :) no need to wait for another lgtm.

**Pull request merged**